cobrowse-sdk-react-native.podspec

```diff
@@ -11,7 +11,7 @@ Pod::Spec.new do |s|
   s.homepage      = package["homepage"]
   s.source        = { :git => 'https://github.com/cobrowseio/cobrowse-sdk-react-native.git' }
   s.platform      = :ios, '11.0'
-  s.dependency      'CobrowseIO/XCFramework', '2.28.6'
```

Comment:
This version just broke our app because it seems like the [version 2.29.6](https://github.com/CocoaPods/Specs/tree/master/Specs/d/a/d/CobrowseIO) doesn't exist.

Am I doing anything wrong here?

Reply:
@uqmessias this was a typo 😬 Please try [`2.18.1`](https://github.com/cobrowseio/cobrowse-sdk-react-native/commit/6002b6110c820b398732fa8e7dae8e60ded384d1) RN release